functions/v2/storage/main.py

```diff
@@ -20,13 +20,13 @@
 
 # Triggered by a change in a storage bucket
 @functions_framework.cloud_event
-def hello_gcs(cloud_event: CloudEvent) -> None:
+def hello_gcs(cloud_event: CloudEvent) -> dict:
     """This function is triggered by a change in a storage bucket.
 
     Args:
         cloud_event: The CloudEvent that triggered this function.
     Returns:
-        None.
+        
```

Comment:
What does it return? Also, this line has spaces in it, which our lint absolutely hates.

Reply:
Forgot to edit this